### PR TITLE
add access keys for the source tabs context menus

### PR DESCRIPTION
**Pull request by bomsy**
Associated Issue: #1624

### Summary of Changes

* Moved access keys to `strings.json` and `debugger.properties`

### Test Plan
- [x] Check to make sure context menus show correct strings

### CODE REVIEW

File: src/components/Editor/Tabs.js [hunk 1]
```diff
@@ -158,23 +161,23 @@ const SourceTabs = React.createClass({
...
       accesskey: "s",
       disabled: false,
       click: () => showSource(tab)
     };
 
     const copySourceUrl = {
       id: "node-menu-close-tabs-to-right",
-      label: "Copy Link Address",
+      label: copyLinkLabel,
       accesskey: "X",
```
Comment (jasonLaster) [inline src/components/Editor/Tabs.js]: actually these keys need to be moved too

Comment (jasonLaster) [inline src/components/Editor/Tabs.js]: screen shot 2017-01-26 at 11 14 38 am

Comment (bomsy) [inline src/components/Editor/Tabs.js]: ooohhh ok!

**Comment by jasonLaster** [comment]
Thanks!

**Pull request merged**